apps/store/src/components/QuickAdd/DismissButton.tsx

```diff
@@ -10,6 +10,7 @@ export const DismissButton = () => {
   const handleDismiss = () => {
     datadogRum.addAction('Quick Add Hide')
     setShow(false)
+    window.scrollTo({ top: 0, behavior: 'smooth' })
```

Comment:
```suggestion
    window.scrollTo({ top: 0 })
```
We already set scroll behaviour globally in [`globalStyles`](https://github.com/HedvigInsurance/racoon/blob/main/packages/ui/src/lib/globalStyles.ts#L31-L39)

<img width="374" alt="image" src="https://github.com/HedvigInsurance/racoon/assets/19200662/5e325d69-727d-40e0-bd6e-c8250133dc18">

and we're taking user's settings into account for that as well. I think if we pass `behavior` to `window.scrollTo` we'd be overriding that, which would make users that opt in for _reduced motion_ still experience some motion here.

Reply:
Ah didn't know that, I'll remove it 🙌